libs/sdk/src/common/notification/notification.ts

```diff
@@ -338,6 +320,16 @@ class NotificationService {
     }
   }
 
+  private getLatestStoredNotificationID() {
+    const localStorageKey = `${this._web3.state.address}-${this.latestSeenNotificationIDKey}`;
```

Comment:
We can extract this `localStorageKey` into a method so it is defined in a single place